tests/rptest/transactions/transactions_test.py

```diff
@@ -481,7 +481,7 @@ def expired_tx_test(self):
             assert False, "tx is expected to be expired"
         except ck.cimpl.KafkaException as e:
             kafka_error = e.args[0]
-            assert kafka_error.code() == ck.cimpl.KafkaError._FENCED
+            assert kafka_error.code() == ck.cimpl.KafkaError.INVALID_TXN_STATE
```

Comment:
is this commit supposed to come after your  (deeper) changes? or is it because of changes in error code mapping commit?

Reply:
yes, i am going to move it, i've put the meat of changes for convenience (easier to ammend)